resources/tests.mjs

```diff
@@ -404,6 +404,32 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "TodoMVC-Lit",
+    url: "todomvc/architecture-examples/lit/rollup/index.html",
+    tags: ["todomvc"],
+    async prepare(page) {
+        await page.wait
+        await page.waitForElement("todo-app");
+    },
+    tests: [
+        new BenchmarkTestStep(`Adding${numberOfItemsToAdd}Items`, (page) => {
+            const todoApp = page.querySelector('todo-app');
+            for (let i = 0; i < numberOfItemsToAdd; i++) {
+                todoApp.dispatchEvent('speedometer-addtodo', {detail: `Something to do ${i}`}, CustomEvent);
```

Comment:
That said I believe the result is similar, because then this dispatches the same events as before under the hood.

Reply:
Ah, I see the argument for how it's done in the readme:

> The speedometer TodoMVC tests typically interact directly with DOM elements, emulating typing into the `<input>` and clicking on buttons and checkboxes, etc. Because Lit uses the Shadow DOM, this code lives in this app and the benchmark code simply dispatches `speedometer-*` events on the `<todo-app>` element, which then calls the same APIs that speedometer usually would to emulate the requested user input.

Thoughts, @rniwa @flashdesignory ?

Reply:
if it's easy to get a reference to the actual input from within the test file, that's preferred maybe. 
I didn't look at the workload yet, but maybe the reason was that it's not that easy to get that reference directly.

Reply:
Discussed this with @flashdesignory who's going to make a suggestion on the best way to handle this (as it's also come up in #249). So this can remain unchanged in the meantime, but the other comments on the PR look ready to address

Reply:
I copied the callElementMethod from your PR @flashdesignory to expose the elements directly. This way the Lit benchmark can otherwise be the same as the other TodoMVC ones.

Reply:
@rictic - if you have time, do you want to try this and let me know if that would work for the lit workload?
https://github.com/WebKit/Speedometer/pull/253

Reply:
@rictic - the pr mentioned above is merged into main

Reply:
That works, but it required some changes to page.querySelectorAll because the DOM is set up like:

* todo-app
  * todo-list
    * todo-item
      - .toggle
      - .destroy
    * todo-item
      - .toggle
      - .destroy
    * ...

The current implementation does the "selectAll" logic only at the final lowest level, so it would only see the first `.toggle` element

Reply:
Can you try this?

Page.querySelectorAll -> todo-items (returns array of PageElements)

PageElement.querySelectorInShadowRoot -> .toggle
PageElement.querySelectorInShadowRoot -> .destroy

Reply:
Here is an example: https://github.com/WebKit/Speedometer/blob/e428accf51b228c1e1c207e0d8edabc518fcd20c/resources/tests.mjs#L178

Reply:
Sure, that works. IMO it's less elegant but it works fine. Updated.